components/dashboard/dashboard.go

```diff
@@ -169,7 +167,7 @@ func (d *Dashboard) ReconcileComponent(ctx context.Context,
 		if platform == deploy.ManagedRhods {
 			if enabled {
 				// first check if the service is up, so prometheus won't fire alerts when it is just startup
-				if err := monitoring.WaitForDeploymentAvailable(ctx, resConf, ComponentNameSupported, dscispec.ApplicationsNamespace, 20, 3); err != nil {
+				if err := monitoring.WaitForDeploymentAvailable(ctx, cli, ComponentNameSupported, dscispec.ApplicationsNamespace, 20, 3); err != nil {
```

Comment:
` 20, 3` - do we really need that configurable for every component? I would assume most just copy-paste the values. Can't we `const` it and simplify this function?